code/setup.dm

```diff
@@ -906,8 +1227,8 @@ SEE_PIXELS	256
 #define PULSE_2FAST		5	//>120 bpm
 #define PULSE_THREADY	6	//occurs during hypovolemic shock
 //feel free to add shit to lists below
-var/list/tachycardics = list("coffee", "inaprovaline", "hyperzine", "nitroglycerin", "thirteenloko", "nicotine")	//increase heart rate
-var/list/bradycardics = list("neurotoxin", "cryoxadone", "clonexadone", "space_drugs", "stoxin")					//decrease heart rate
+var/list/tachycardics = list(COFFEE, INAPROVALINE, HYPERZINE, NITROGLYCERIN, THIRTEENLOKO, NICOTINE)	//increase heart rate
+var/list/bradycardics = list(NEUROTOXIN, CRYOXADONE, CLONEXADONE, "space_drugs", STOXIN)					//decrease heart rate
```

Comment:
The kind of stuff that really tells you why we needed this in the first place.
I'm tempted to fix it, but it might violate atomicity.